### DISCUSSION TITLE
1.23.1

### DISCUSSION THREAD

**Discussion by dani-garcia** [dani-garcia/vaultwarden#2150] [General]
⚠️  **Reminder**: If you are still using the `bitwardenrs/server*` Docker images, you need to migrate to the new `vaultwarden` image. Check https://github.com/dani-garcia/vaultwarden/discussions/1642 for an explanation. The old images will not receive any new updates any longer.

## What's Changed
* Add email notifications for incomplete 2FA logins by @jjlin in https://github.com/dani-garcia/vaultwarden/pull/2067
* Fix conflict resolution logic for `read_only` and `hide_passwords` flags by @jjlin in https://github.com/dani-garcia/vaultwarden/pull/2073
* Fix missing encrypted key after emergency access reject by @jjlin in https://github.com/dani-garcia/vaultwarden/pull/2078
* Fix PostgreSQL migration by @jjlin in https://github.com/dani-garcia/vaultwarden/pull/2080
* Macro recursion decrease and other optimizations by @BlackDex in https://github.com/dani-garcia/vaultwarden/pull/2084
* Enabled trust-dns and some updates. by @BlackDex in https://github.com/dani-garcia/vaultwarden/pull/2125
* Update web vault to 2.25.0

**Full Changelog**: https://github.com/dani-garcia/vaultwarden/compare/1.23.0...1.23.1

<hr /><em>This discussion was created from the release <a href='https://github.com/dani-garcia/vaultwarden/releases/tag/1.23.1'>1.23.1</a>.</em>

**Comment by kangaroo72** [comment]
Thank you - I waited a long time 👍

**Comment by MilesTEG1** [comment]
Hello,
How can we set the trust-dns value ? I don' t find any indication in the wiki...
Thanks in advance :)

**Reply by BlackDex** [reply]
You can't set it. It is used by default by reqwest.

**Reply by MilesTEG1** [reply]
Ok, thanks for your answer.
Is there any way to see what value it gets (automatically) ?

**Reply by BlackDex** [reply]
In what sense?
You can enable `LOG_LEVEL=debug` which will also show the debugging messages from `trust-dns-resolver`

**Reply by MilesTEG1** [reply]
> In what sense? You can enable `LOG_LEVEL=debug` which will also show the debugging messages from `trust-dns-resolver`

It's set to warn.
I'll try set it to debug to see the debug info of trust-dns-resolver.

**Comment by rezzorix** [comment]
@dani-garcia Just a small thought: would it be possible to add the Vaultwarden version on the webinterface, too?

Maybe like this with VW version in brackets?
![image](https://user-images.githubusercontent.com/72935517/147185157-e6e3be2f-9f12-482a-9911-5ac27d63f0c6.png)

**Reply by BlackDex** [reply]
That isn't possible for multiple reasons.
- the web-vault is independent from the backend and can be build separately.
- we do not change these files at runtime, all text is embedded and static.

If you want to know the version of Vaultwarden you are running you can go the /admin/diagnostics page which will show you the versions.
